### PR TITLE
fix: Re-add WebSocket ping

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/api-client/src/main/tcp/ReconnectingWebsocket.ts [hunk 1]
```diff
@@ -39,7 +39,15 @@ export enum WEBSOCKET_STATE {
...
 
+export enum PingMessage {
+  PING = 'ping',
+  PONG = 'pong',
+}
+
 export class ReconnectingWebsocket {
+  private static readonly CONFIG = {
+    PING_INTERVAL: TimeUtil.TimeInMillis.SECOND * 20,
+  };
```
Comment (ffflorian) [inline packages/api-client/src/main/tcp/ReconnectingWebsocket.ts]: Why not move this outside of the class? It's static and private.

File: packages/api-client/src/main/tcp/ReconnectingWebsocket.ts [hunk 2]
```diff
@@ -118,6 +139,30 @@ export class ReconnectingWebsocket {
     }
   }
 
+  private startPinging(): void {
+    this.stopPinging();
```
Comment (ffflorian) [inline packages/api-client/src/main/tcp/ReconnectingWebsocket.ts]: Stops pinging after the start?

Comment (Yserz) [reply packages/api-client/src/main/tcp/ReconnectingWebsocket.ts]: No, that's stopping before starting it. There are cases where we had 2 pingers running, that's why the failed ping check was actually failing.

File: packages/api-client/src/main/tcp/ReconnectingWebsocket.ts (hunk 1 above)
Comment (Yserz) [reply packages/api-client/src/main/tcp/ReconnectingWebsocket.ts]: Because it's private to `ReconnectingWebsocket` more than to the file.

**Pull request merged**